### ISSUE TITLE
Tech revamp.

### ISSUE THREAD

**Issue by cash1981** [cash1981/playciv#168]
Todays tech tree: 

<img width="622" height="213" alt="Image" src="https://github.com/user-attachments/assets/263a60fb-65fd-446c-b27e-0b7f62aa0847" />

Changes:

1. Create a tech tree with tech cards

Instead of having the boring tech tree with text only I want to build it using real tech card images. 
It must upload the same pyramid style where level 1 is at the bottom, then level 2, level 3 and so on.

The last tech level 5 card space flight must also appear

2. I want users to be able to see the techs cards they want to research and read what they do and see the cards. 
I want to group all level 1, 2, 3 and 4 and 5 card images, and be able to view when to see what I want to pick instead of the combo box we have today. 
So I think you group them by levels, perhaps a tab for each level, then you can browse and read the card and choose it. In addition to the card image it self there should be text too to make it accessible.
